cli_sdk.py

```diff
@@ -108,10 +108,12 @@ def get_token_coverage(intent, transcript):
     token_lines = list(get_token_logs(intent, transcript))[3:-1]
     tokens = [[w for w in t.split(" ") if w][-2:] for t in token_lines]
     transcript = [w[-1] for w in tokens]
-    coverage = [("*" if w[0] != "None" else " ")*len(w[-1]) for w in tokens]
-    print()
-    print("Recognized parts of the transcript are denoted with '*'")
-    print()
-    print(' '.join(transcript))
-    print(' '.join(coverage))
-    print()
+    coverage = [("*" if w[0] != "None" else " ") * len(w[-1]) for w in tokens]
+    print("""
+    
+    Recognized parts of the transcript are denoted with '*'
+    
+    {0}
+    {1}
+    
+    """.format(' '.join(transcript), ' '.join(coverage)))
```

Comment:
Do you want everything to have and indent? If not we need to write.

```python
    print("""

Recognized parts of the transcript are denoted with '*'
    
{0}
{1}
 
""".format(' '.join(transcript), ' '.join(coverage)))
```

Reply:
I see your comments above.

Reply:
The indent makes it a bit more readable for me in code, and looks fine in the cli since it's at the same level as the interpreter prompt `>>>`. So, I kept it in.